elastic/kibana
# [Console] Escape single quotes surrounded by double quotes

## Summary

Fix https://github.com/elastic/kibana/issues/49235

## How to test

1. See the original issue for steps.
2. (covered by unit tests) Create a number of different queries and click around between them (preferably with JSON bodies). Keep an eye on the console for any errors (there should be none)
3. (covered by unit tests) Try copying the default query to cURL, it should still work as before

## Release note

We fixed a bug with Console's Copy As cURL functionality that would not properly escape single quotes in JSON string values which created issues with copying SQL queries from Console to cURL.

### Checklist
